index.html

```diff
@@ -103,17 +103,25 @@ <h3 class="skills-title">Skills</h3>
   
 <section class="my-works-section">
     <div class="my-works-titles">
-      <i class="fa fa-arrow-down my-work-icon"></i>
+      <i class="fa fa-arrow-down my-work-icon"></i>      
       <h3 class="my-works-section-title">Work</h3>
     </div>
 
+    <!-- <div class="my-works-titles">
+      <section class="my-arrow-down">
+        <img src="./images/vector_5.svg" alt="my work image vector 5">
+        <img src="./images/vector_6.svg" alt="my work image vector 6">
+      </section>
+      <h3 class="my-works-section-title">Work</h3>
+
+    </div> -->
```

Comment:
- [ ] Please make sure to remove any unnecessary code from your codebase, as this is part of the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) (and it's also more professional :zany_face:)